things/calculator/mqtt/js/main.js

```diff
@@ -5,8 +5,8 @@ const path = require('path')
 const { JsonPlaceholderReplacer } = require('json-placeholder-replacer')
 require('dotenv').config()
 
-const hostname = 'test.mosquitto.org'
-let portNumber = 1883
+const hostname = process.env.HOSTNAME ?? 'test.mosquitto.org'
```

Comment:
this should be a separate thing than hostname. we may host a broker but I doubt it

Reply:
Yes that would make things more coherent